src/NServiceBus.AcceptanceTesting/NServiceBus.AcceptanceTesting.csproj

```diff
@@ -20,4 +20,8 @@
     <Description>Acceptance testing framework for NServiceBus endpoints. This is an unsupported package.</Description>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectCapability Remove="TestContainer" />
```

Comment:
that seems like some accident we should revert?

Reply:
No, this was my intentional change that I was talking about in the last sync meeting.

Reply:
ah ok, I missed that the element name was `ProjectCapability`. Ok that makes sense. Maybe we can add a comment to clarify a little bit why this was added?